debian_package_manager/internal/build/bazel_test.go

```diff
@@ -0,0 +1,117 @@
+package build
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/GoogleContainerTools/distroless/debian_package_manager/internal/deb"
+	"github.com/google/go-cmp/cmp"
+)
+
+func testdata() map[string]map[string]map[string]*deb.Package {
```

Comment:
`map[string]map[string]map[string]*deb.Package` is wild lol

Reply:
This makes printing versions pretty easy (which may not be needed once we go full sbom? then we can just add a sorting function to deb.Package and we should get reproducible config generation).